decidim-proposals/app/views/decidim/proposals/admin/proposals/_proposal-tr.html.erb

```diff
@@ -0,0 +1,63 @@
+<tr data-id="<%= proposal.id %>">
+  <td>
+    <%= check_box_tag "proposal_ids_s[]", proposal.id, false, class: "js-check-all-proposal js-proposal-list-check  js-proposal-id-#{proposal.id}" %><br />
```

Comment:
Is this `proposal_ids_s` (the `_s` part) intended? Or is this a typo?

Reply:
It's intended, 
these `checkbox`s are used to link the visible (clicked by the user) with the hidden ones sent by the `form` named `proposal_ids`,
I'll rename them to `proposal_ids_tr`, ok?